### ISSUE TITLE
[FORMATTING] GBQ @@error.message not supported

### ISSUE THREAD

**Issue by crazy142857** [sql-formatter-org/sql-formatter#567]
```
BEGIN
select 1/0;
EXCEPTION WHEN ERROR THEN
SELECT @@error.message;
END;
```
Unable to format SQL: Error: Parse error: Unexpected "@@error.me" at line 4 column 8

This code is for BigQuery.

**Comment by nene** [comment]
Thanks for reporting.

Looks like in BigQuery these `@@something` are system variables, which the formatter currently doesn't support: https://cloud.google.com/bigquery/docs/reference/system-variables

**Comment by nene** [comment]
This is now fixed in 12.1.2 release.
